### ISSUE TITLE
Not sure about ports

### ISSUE THREAD

**Issue by mouzzampk2014** [romancin/rutorrent-flood-docker#167]
Hi, could you please help me out what ports I need and why, just trying to understand :)
```
-p 3000:3000 
-p 51415-51415:51415-51415
```
Thank you please

**Comment by BoKKeR** [comment]
from the readme

Map any local port to 80 for rutorrent access (Default username/password is admin/admin)
Map any local port to 443 for SSL rutorrent access if SSL_ENABLED=yes (Default username/password is admin/admin)
Map any local port to 51415 for rtorrent
Map any local port to 3000 for SSL flood access

**Comment by mouzzampk2014** [comment]
Yes this one is for gui `80` and this one for network port range `51415`?

**Comment by BoKKeR** [comment]
51415 its for rtorrent client itself as far I know, if you want to connect to it directly

**Comment by mouzzampk2014** [comment]
Hmm okay let me test it. Thanks again

**Comment by KaKi87** [comment]
What about HTTP (non-SSL) flood access? Thanks
